### ISSUE TITLE
Jinja2 support

### ISSUE THREAD

**Issue by jorenham** [praekelt/django-ultracache#5]
Is fragment ultracaching also supported in jinja2?

**Comment by hedleyroos** [comment]
No, it is not. Adding support would likely not be that hard since Jinja2 is also single threaded, but since the speed advantage of Jinja2 vs Django templates is somewhat mitigated by ultracache I've never used it.

**Comment by jorenham** [comment]
Then consider it a feature request :smile:
